pkg/meta/config.go

```diff
@@ -55,3 +62,34 @@ func (f *Format) RemoveSecret() {
 		f.EncryptKey = "removed"
 	}
 }
+
+func (f *Format) CheckVersion() bool {
+	if f.MetaVersion > 1 {
+		return false
+	}
+
+	if f.ClientVersions == "" {
+		return true
+	}
+	ps := strings.Fields(f.ClientVersions)
+	if len(ps) == 0 {
+		return true
+	}
+
+	var ok bool
+	if r, err := version.Compare(ps[0]); err == nil {
+		ok = r >= 0
+	} else {
+		logger.Errorf("Compare versions: %s", err)
+		return false
+	}
+	if ok && len(ps) > 1 {
+		if r, err := version.Compare(ps[1]); err == nil {
+			ok = r <= 0
+		} else {
+			logger.Errorf("Compare versions: %s", err)
+			return false
+		}
+	}
+	return ok
```

Comment:
Add meanlingful logging if it not matched

Reply:
Or, return the error message

Reply:
same as above